device-communication/README.md

```diff
@@ -0,0 +1,104 @@
+# Device Communication API
+
+Device communication API enables users and applications to send configurations and commands to devices via HTTP
+endpoints.
+
+![img.png](img.png)
+
+### Application
+
+The application is reactive and uses Quarkus Framework for the application and Vertx tools for the HTTP server.
+
+### Hono internal communication
+
+API uses [Google's PubSub](https://cloud.google.com/pubsub/docs/overview?hl=de) service to communicate with the command
+router.
+
+## API endpoints
+
+#### commands/{tenantId}/{deviceId}
+
+- POST : post a command for a specific device
```

Comment:
A hint that this isn't implemented yet would be good here.

Reply:
Done